.github/workflows/create-security-patch-from-security-mirror.yml

```
@@ -11,6 +11,7 @@ on:
     branches:
       - "main"
       - "v*.*.*"
+      - "release-*"
```

```suggestion
      - "release-*.*.*"
```
Nit: this could help avoid triggering this for non-release branches.